### DISCUSSION TITLE
Welcome to Team-Rio-Laravel Discussions!

### DISCUSSION THREAD

**Discussion by victorazangu** [SpaceyaTech/Team-Rio-Laravel#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by victorazangu** [comment]
was hoping we should start implementing some features

**Comment by thomi** [comment]
Hi was wondering is there a system overview / architecture or schema design we should adhere to? Since we're building a db agnostic application designing a schema seems like a prudent thing to do before we start any coding?  


Would love to hear any thoughts on this and how you guys design your apps even before coding begins!

**Reply by victorazangu** [reply]
we have documentation https://github.com/SpaceyaTech/CMS-Backend-Repository/blob/main/SYTcms.mwb
go through it and you can see the entities/models that we discuss during our backend meetings. also on Tuesdays and Thursdays, we have  backend meetings I highly encourage  not to miss they are very important
